src/components/EnvironmentOverride/DeploymentTemplateOverride.tsx

```diff
@@ -203,6 +203,7 @@ function DeploymentTemplateOverrideForm({state, handleOverride, dispatch, initia
             setLoading(not)
         }
     }
+    const appMetricsEnvironmentVariableEnabled = process.env.NODE_ENV === 'production' && window._env_ && window._env_.APPLICATION_METRICS_ENABLED;
```

Comment:
process.env.NODE_ENV and APPLICATION_METRICS_ENABLED are two independent flag. 
is `APPLICATION_METRICS_ENABLED`  flag dependent on `NODE_ENV`?

Reply:
yeah they are independent. the implementation is kept similar to other env variables. For development purpose this can be changed as needed.

Alternatively, this can be made to 

const appMetricsEnvironmentVariableEnabled = window._env_ && window._env_.APPLICATION_METRICS_ENABLED;

Would it be okay?

Reply:
yes you can change it to 

`const appMetricsEnvironmentVariableEnabled = window.env && window.env.APPLICATION_METRICS_ENABLED;`

Reply:
done and pushed